### ISSUE TITLE
'@import (reference)' Bug

### ISSUE THREAD

**Issue by baijunjie** [less/less.js#3071]
keyframes.less
```css
.keyframes(@animation_name) {
    @keyframes @animation_name {
        from { opacity: 0; }
        to   { opacity: 1; }
    }
}
```

styles.less
```css
@import "keyframes.less";
.keyframes(opacity-animation);
```
output:
```css
@keyframes opacity-animation {
  from {
    opacity: 0;
  }
  to {
    opacity: 1;
  }
}
```

styles.less
```css
@import (reference) "keyframes.less";
.keyframes(opacity-animation);
```
output:
```css
// empty ???
```

**Comment by seven-phases-max** [comment]
Update your Less version - it's fixed in `v2.3.0`(?). 

Either way try to not use `reference` where it's not *absolutely unavoidable* (the feature is hackish in its very nature and there's infinite amount of issues with it in many edge cases - probably some of them will be fixed evenually - but I can't see anyone is actually going to do that any time soon).
So, for instance, if `"keyframes.less"` is mean to be a reusable component (thus containing only really reusable stuff like parametric mixins, variables and so on) it should never require to be imported via `reference`.

**Comment by seven-phases-max** [comment]
Closing as fixed in recent versions.
